functions/go/render-helm-chart/README.md

```diff
@@ -100,7 +100,7 @@ helmCharts:
   valuesInline: map[string]interface{}
   valuesFile: string
   valuesMerge: string
-  includeCRDs: bool
+  includeCRDs: string
```

Comment:
Why change this?

Reply:
The include crds example was failing, saying that it cannot marshal string to bool

Reply:
Changing it back to bool and for it to marshal to map[string]string and copy all the fields